### PR TITLE
Update job template url

### PR DESCRIPTION
**Pull request by sebastienvas**
```release-note
NONE
```

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 plank:
-  job_url_template: 'https://k8s-gubernator.appspot.com/build/istio-prow/{{if eq .Spec.Type "presubmit"}}pull{{else if eq .Spec.Type "batch"}}pr-logs/pull{{else}}.{{end}}{{if ne .Spec.Refs.Org ""}}{{if ne .Spec.Refs.Org "kubernetes"}}/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}{{else if ne .Spec.Refs.Repo "kubernetes"}}/{{.Spec.Refs.Repo}}{{end}}{{end}}{{if eq .Spec.Type "presubmit"}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else if eq .Spec.Type "batch"}}/batch{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
+  job_url_template: 'https://k8s-gubernator.appspot.com/build/istio-prow/{{if eq .Spec.Type "presubmit"}}pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{else }}.{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
```
Comment (nclando) [inline prow/config.yaml]: add {{end}} here?

**Review by nlandolfi — CHANGES_REQUESTED** [review]
one missing end block

**Review by nlandolfi — APPROVED** [review]
LGTM

**Pull request merged**